sites/partners/src/pages/listings/[id]/index.tsx

```diff
@@ -88,6 +89,7 @@ export default function ListingDetail(props: ListingProps) {
                 <div className="flex flex-row flex-wrap ">
                   <div className="info-card md:w-9/12 overflow-hidden">
                     <DetailListingData />
+                    <DetailListingNotes />
```

Comment:
I noticed the DetailListingData is not in the figma file, did we get confirmation that the notes should be below that?

Reply:
Confirmed w product to keep current behavior